kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -237,6 +244,7 @@ async def get(self, *, entries: Iterable[EntryRequest], **rpc_kwargs) -> List[Da
             rpc_kwargs
                 grpc.*MultiCallable kwargs e.g. timeout, metadata, credentials.
         """
+        rpc_kwargs.__setitem__("metadata", await self.generate_metadata_header(rpc_kwargs.get("metadata")))
```

Comment:
I don't see any reason to call the dunder method explicitly.
```
rpc_kwargs["metadata"] = ...
```
is both shorter and IMO more idiomatic.

Applies to all the instances of this.